server/chatserver.h

```diff
@@ -18,6 +18,6 @@ private slots:
     void socketDisconnected();
 
 private:
-    QWebSocketServer* m_webSocketServer;
-    QList<QWebSocket*> m_clients;
+    std::unique_ptr<QWebSocketServer> m_webSocketServer;
+    std::vector<std::unique_ptr<QWebSocket*>> m_clients;
```

Comment:
I think we shouldn't use std::vector<std::unique_ptr to raw pointers, this may cause memory problems

Reply:
Do you have other idea? We have to store QWebSocket*, there is no option to store as a normal type